packages/buildbucket-dart/pubspec.yaml

```diff
@@ -7,11 +7,11 @@ environment:
   sdk: '>=3.0.0-0 <4.0.0'
 
 dependencies:
-  protoc_plugin: 21.1.2
-  fixnum: 1.1.0
-  protobuf: 3.1.0
-  grpc: 3.2.4
+  protoc_plugin: ^21.1.2
+  fixnum: ^1.1.0
+  protobuf: ^3.1.0
+  grpc: ^3.2.4
```

Comment:
Any reason to use the undetermined caret version? We have been trying to use fixed versions to avoid potential breakage.